SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -621,6 +621,10 @@ theorem allOnes_shiftLeft_or_shiftLeft {x : BitVec w} (n : Nat) :
     BitVec.allOnes w <<< n ||| x <<< n = BitVec.allOnes w <<< n := by
   simp [← BitVec.shiftLeft_or_distrib]
 
+@[simp]
+theorem zero_sub {x : BitVec w} : 0 - x = - x := by
+    simp [bv_toNat]
```

Comment:
```suggestion
theorem zero_sub {x : BitVec w} : 0 - x = - x := by
    simp [bv_toNat]
```
```suggestion
theorem zero_sub {x : BitVec w} : 0#w - x = - x := by
    simp [bv_toNat]
```

And we probably want to have the simp-normal-form for zero bitvectors, where the bitwidth is explicit.

Reply:
thanks for the comments!